index.html

```diff
@@ -5,9 +5,61 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
-    <title>Hello Microverse</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39+Text&family=Poppins:wght@200;400;600&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300;1,400;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39+Text&family=Playfair+Display&family=Poppins:wght@200;400;600&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300;1,400;1,500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://www.pngplay.com/wp-content/uploads/12/Astro-Boy-Transparent-Images.png"
+    />
+    <title>AstroboyReloaded | Portfolio</title>
   </head>
   <body>
-    <h1 class="title">Hello Microverse!</h1>
+    <header>
+      <nav>
+        <div
+          class="logo"
+          role="img"
+          aria-label="Logo: Right Triangle and rhombus"
+        ></div>
+        <label class='menu-btn-container'>
+          <input id="menu-toggle" type="checkbox" />
+          <div class='menu-btn'></div>
+        </label>
+      </nav>
+      <!-- headline -->
+      <section class="headline-container">
+        <h1 id="headline-title" class="headline-title playfair">Alex Muñoz</h1>
+        <p class="headline-paragraph poppins">
+          &nbspHey, there! I am a software developer! I can help you build a
+          product, feature or website. Take a look at my work. If you like what
+          you see and have a project you would like coded, don’t hesitate to
+          contact me.
+        </p>
+        <a href="#" class="btnB start-colaboration poppins">
```

Comment:
- [ ] Nice work @astroboyReloaded, you have done a great job so far, but we have an issue with your images, they are not displaying, see this below and make sure you fix this:
- 
![5](https://user-images.githubusercontent.com/77843050/211936706-2b2a39d8-8ef3-4ebc-9a34-1035c3cd12e7.gif)